### ISSUE TITLE
low accuracy using Resnet50 with dataset UCF101 without pre-trained model

### ISSUE THREAD

**Issue by SwordHarry** [okankop/Efficient-3DCNNs#44]
```
python main.py --root_path ./ \
	--video_path ./ucf101_dataset/ucf101_jpg \
	--annotation_path Efficient-3DCNNs/annotation_UCF101/ucf101_01.json \
	--result_path ./results/ucf101/resnet50/ \
	--dataset ucf101 \
	--n_classes 101 \
	--ft_portion complete \
	--model resnet --model_depth 50 --resnet_shorcut B\
	--groups 3 \
	--width_mult 0.5 \
	--train_crop random \
	--learning_rate 0.1 \
	--sample_duration 16 \
	--downsample 1 \
	--batch_size 64 \
	--n_threads 16 \
	--checkpoint 1 \
	--n_val_samples 1 \
        --test
```
After I run the code for training and validating the resnet50 without pretrain model, the top-1 acc is about 40 and top-5 acc is about 65; but in the paper, acc of resnet-50 with dataset UCF101 is 88.92
 how can i reach the acc in the paper? Or do I have to load the pre-trained model?

**Comment by poincarelee** [comment]
@SwordHarry  how many epochs have you run? the default 250?  Have you tried learining rate other than 0.1? 
![efficient-3DCNN](https://user-images.githubusercontent.com/38481402/178683995-76558d42-0783-4e20-93d6-53dd8487e008.png)

this is my training process, the precision and loss is vibrating with one epoch, seems wierd. Have you met this before?
